GeosCore/carbon_gases_mod.F90

```diff
@@ -166,7 +166,13 @@ SUBROUTINE Emiss_Carbon_Gases( Input_Opt,  State_Chm, State_Diag,            &
     !========================================================================
     ! CO2 production from CO oxidation
     !========================================================================
-    IF ( Input_Opt%LCHEMCO2 .and. id_CO2_adv > 0 ) THEN
+
+    ! If CO2 is an advected species but CO is not, use CO2_COPROD field
+    ! from HEMCO. Otherwise compute CO2 production via KPP.
+    IF ( id_CO2_adv > 0 .and. id_CO_adv <= 0 ) THEN
+
+       WRITE( 6, 100 )
```

Comment:
@msulprizio: I think we should wrap the write statement in a `IF ( Input_Opt%amIRoot ) THEN ...` block.

Reply:
Also not sure if that has to be written out every timestep or just on the 1st timestep.  I will defer to @lizziel.

Reply:
I agree to wrap in amIRoot and also to only print in first timestep

Reply:
This has now been addressed in commit https://github.com/geoschem/geos-chem/pull/2510/commits/68604792ea1f4340f37d1d33bf1782c16e48e795.